tests/security/apparmor/aa_logprof.pm

```diff
@@ -26,21 +26,33 @@
 # - Check if nscd could start with the temporary apparmor profiles
 # - Cleanup temporary directory
 # Maintainer: llzhao <llzhao@suse.com>
-# Tags: poo#36892, poo#45803
+# Tags: poo#36892, poo#45803, poo#81730, tc#1767574
 
 use base "apparmortest";
 use strict;
 use warnings;
 use testapi;
 use utils;
 use version_utils 'is_tumbleweed';
+use version_utils qw(is_tumbleweed is_sle);
```

Comment:
Just a small question, is it possible to remove the last line? 
`use version_utils 'is_tumbleweed';`